.buildkite/bk.integration.pipeline.yml

```diff
@@ -28,6 +28,79 @@ steps:
       image: "docker.elastic.co/ci-agent-images/platform-ingest/buildkite-agent-beats-ci-with-hooks:0.5"
       useCustomGlobalHooks: true
 
+  - label: Start Serverless project
+    key: integration-serverless
+    concurrency_group: elastic-agent-extended-testing/serverless-integration
```

Comment:
@pazone this concurrency_group regards only this step or the full pipeline? in other words as soon as this step end a concurrency slot will be freed or it will wait for the whole pipeline to end until freeing one?